File: Shop/src/app/components/orders/Views/OrderView.ts
export class OrderView {
  Id: number;
  CustomerName: string;
  CustomerAddress: string;
  TotalCost: number;
  Status: string;

  constructor(Id: number, CustomerName: string, CustomerAddress: string, TotalCost: number, Status: string) {
    this.Id = Id;
    this.CustomerName = CustomerName;
    this.CustomerAddress = CustomerAddress;
    this.TotalCost = TotalCost;
    this.Status = Status;
  }
}

File: WebShop/WebShop/DTO/Views/OrderView.cs
namespace WebShop.DTO.Views
{
    public class OrderView
    {
        public int Id { get; set; }
        public string CustomerName { get; set; }
        public string CustomerAddress { get; set; }
        public double TotalCost { get; set; }
        public string Status { get; set; }
    }
}
